modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -64,6 +64,27 @@ Before you can use Anypoint Service Mesh, you must have the following roles and
 For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster
 
+== Ports, IPs and Hostnames Whitelist Requirements
+
+In your network, you might need to whitelist the hostnames and ports of external resources to allow Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
+
+[%header%autowidth.spread,cols="3+a"]
+|===
+|Plane |Host |Port
+|*US*|*anypoint.mulesoft.com* | 443
+|*US*|*analytics-ingest.anypoint.mulesoft.com* |  443
+|*US*|**.dkr.ecr.us-east-1.amazonaws.com* |  443
+|*US*|*exchange-files.anypoint.mulesoft.com* |  443
+|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443
+|*EU*|*eu1.anypoint.mulesoft.com* | 443
+|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443
+|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443
```

Comment:
I'm not sure what the double asterisk here and on line 76 mean, but I assume they're intentional.

Reply:
It's fine, we need to whitelist any subdomain under `dkr.ecr.eu-central-1.amazonaws.com` so `*.dkr.ecr.eu-central-1.amazonaws.com` it's ok IMO.